ipatests/test_integration/test_membermanager.py

```diff
@@ -212,3 +212,51 @@ def test_hostgroup_member_manager_nopermission(self):
             "'write' privilege to the 'memberManager' attribute of entry"
         )
         assert expected in result.stdout_text
+
+    def test_member_manager_upgrade_scenario(self):
+        user = "idmuser"
+        password = "Secret123"
+        group1 = "role-groupmanager"
+        group2 = "role-usergroup-A"
+
+        master = self.master
+
+        tasks.group_add(master, group1)
+        tasks.group_add(master, group2)
+        tasks.create_active_user(master, user, password)
+
+        tasks.kinit_admin(master)
+        master.run_command(["ipa", "group-add-member", group1, "--user", user])
+        master.run_command(["ipa", "group-add-member-manager", "--groups",
+                            group1, group2])
+
+        ldif_text = """dn: cn=hostgroups,cn=accounts,dc=ipa,dc=test
```

Comment:
it will fail to recognize if relam is used other than `ipa.test`.
consider replacing it with something like https://github.com/freeipa/freeipa/blob/master/ipatests/test_integration/test_replica_promotion.py#L534

Reply:
Replaced, thank you for this clear information also with link.